server/sonar-server/src/main/java/org/sonar/server/computation/task/projectanalysis/step/NewSizeMeasuresStep.java

```diff
@@ -93,45 +97,60 @@ public void execute() {
       FormulaExecutorComponentVisitor.newBuilder(metricRepository, measureRepository)
         .withVariationSupport(periodsHolder)
         .buildFor(ImmutableList.of(duplicationFormula)))
-      .visit(treeRootHolder.getRoot());
+          .visit(treeRootHolder.getRoot());
   }
 
-  private static class NewDuplicationCounter implements Counter<NewDuplicationCounter> {
+  private static class NewSizeCounter implements Counter<NewSizeCounter> {
+    private final MeasureRepository measureRepository;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "measureRepository" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1068)